Geometry/MTDCommonData/test/DD4hep_TestMTDIdealGeometry.cc

```diff
@@ -31,6 +31,7 @@
 
 #include "DataFormats/Math/interface/GeantUnits.h"
```

Comment:
Can line 32 be deleted?

Reply:
Also, change `BuildFile.xml` to delete `DataFormats/Math` and add DD4hep?

Reply:
```DataFormats/Math/interface/GeantUnits.h``` in itself is no more strictly needed, but ```DataFormats/Math/interface/angle_units.h``` that was included in it is still needed (and therefore it should not be removed from the BuildFile)